src/lib/learn-client/api/tutorial/index.ts

```diff
@@ -39,6 +39,17 @@ export async function getTutorial(
 }
 
 // getTutorials
```

Comment:
❓ : I like this for narrowing down types 💯 Would it make sense to change this comment to read `// getTutorials type safeguards` or similar, to denote that this series of functions is specifically for narrowing down `getTutorials`'s return type?